google/_modules/gke/node_pool/variables.tf

```diff
@@ -115,3 +115,13 @@ variable "node_locations" {
   description = "List of zones in the cluster's region to start worker nodes in. Defaults to cluster's node locations."
   default     = null
 }
+
+variable "guest_accelerator" {
+  type        = map(any)
+  description = <<EOF
+`guest_accelerator` block supports during node_group creation, useful to provision GPU-capable nodes. Default to `null` or `{}` which will disable GPUs.
+
+Please check https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#nested_guest_accelerator for more information.
```

Comment:
Can we remove this please check and the url. One line description above should be good enough.